composer.json

```diff
@@ -4,6 +4,6 @@
     "type": "drupal-module",
     "license": "GPL-2.0-or-later",
     "require": {
-        "drupal/core": "^8 || ^9"
+        "drupal/core": "^10.0"
```

Comment:
lets support ^8 || ^9 || ^10

Non of our changes seem to remove support for < Drupal 10 version